lang/cs/Org.Apache.REEF.IMRU/OnREEF/IMRUTasks/TaskHostBase.cs

```diff
@@ -157,7 +157,7 @@ public void OnNext(ICloseEvent closeEvent)
         /// <summary>
         /// Dispose function. Dispose IGroupCommunicationsClient.
         /// </summary>
-        public void Dispose()
+        public virtual void Dispose()
```

Comment:
The Dispose() defined in IDisposable is not virtual.  Is there any issue here? 

http://stackoverflow.com/questions/3619490/why-should-dispose-be-non-virtual